hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/contracts/services/AppendRequestParameters.java

```diff
@@ -72,4 +75,12 @@ public boolean isAppendBlob() {
   public String getLeaseId() {
     return this.leaseId;
   }
+
+  public boolean getIsExpectHeaderEnabled() {
```

Comment:
change to `isExpectHeaderEnabled()` for consistency with the AbfsOutputStreamContext property